src/main/java/org/ecocean/Decision.java

```diff
@@ -165,6 +164,103 @@ public static String findWinner(JSONObject winningIndividualTracker) {
       return currentWinner;
     }
 
+    public static List<String> getEncounterIdsOfMostAgreedUponMatches(List<Decision> decisionsForEncounter){
+      List<String> matchedIds = new ArrayList<String>();
+      String currentMatchedMarkedIndividualId = null;
+      Double currentMatchedMarkedIndividualCounter = 0.0;
+      JSONObject winningIndividualTracker = new JSONObject();
+      JSONObject currentDecisionValue = new JSONObject();
+      if(decisionsForEncounter!=null && decisionsForEncounter.size()>0){
```

Comment:
i feel like this chunk is very redundant in this class... the part where it iterates over decisions and find the ones weighted most.  i think i counted 3 times its in there now.  maybe pop the guts of this into its own class-method for them all to use?